packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.tsx

```diff
@@ -29,6 +28,8 @@ import { useBoxedExpressionEditorI18n } from "../../i18n";
 import { useBoxedExpressionEditorDispatch } from "../BoxedExpressionEditor/BoxedExpressionEditorContext";
 import { DataTypeSelector } from "../ExpressionDefinitionHeaderMenu";
 import "./ParametersPopover.css";
+import { Button, EmptyState, EmptyStateIcon, Title } from "@patternfly/react-core";
+import { CubesIcon } from "@patternfly/react-icons";
```

Comment:
We should use most specific import as we did for 
`import { Button } from "@patternfly/react-core/dist/js/components/Button"`